fiscal_epos_print/models/point_of_sale.py

```diff
@@ -0,0 +1,16 @@
+from odoo import fields, models
+
+
+class PosConfig(models.Model):
+    _inherit = 'pos.config'
+
+    printer_ip = fields.Char(
+        'Printer IP Address',
+        help='The hostname or ip address of the hardware printer, '
+        'please fill this field if you want use you receipts printer',
+        size=45
+    )
+    use_https = fields.Boolean(
+        string='Use https',
+        default=False,
+    )
```

Comment:
Aggiungerei anche il seriale della stampante )da vedere come si potrebbe acquisire in automatico)

Reply:
Da usare poi per i resi?

Reply:
Di sicuro nei resi, non so per le fatture

Reply:
Considera che negli scontrini RT le informazioni per il reso sono già presenti, incluso il seriale

Reply:
Scusa ma non ho capito cosa intendi

Reply:
Operativamente il cliente si presenta con lo scontrino di vendita, su cui sono stampati i dati di quello scontrino, da usare per il reso: data, numero chiusura, numero scontrino, seriale stampante.
Quindi l'operatore POS non deve far altro che inserirli leggendoli dallo scontrino

Reply:
Capito, io l'avevo pensata in modo differente per evitare all'operatore di inserire tutti i dati. Dati tra l'altro che al momento non sono memorizzati da nessuna parte. Comunque come scrivevo in chat, sarebbe meglio fare una call.

Reply:
Seriale aggiunto